addons/mail/static/src/owl/components/chatter_topbar/chatter_topbar.js

```diff
@@ -28,15 +29,68 @@ class ChatterTopbar extends Component {
 
     /**
      * @private
-     * @param {Event} ev
+     * @param {MouseEvent} ev
      */
     _onClickAttachments(ev) {
         this.trigger('o-chatter-topbar-select-attachment');
     }
+
+    /**
+     * @private
+     * @param {MouseEvent} ev
+     */
+    _onClickFollow(ev) {
+        this.trigger('o-chatter-topbar-follow');
```

Comment:
unhandled event for the moment

Reply:
Yep, followers etc handling is not yet done